### PR TITLE
Add   walnutcreek file

### PR DESCRIPTION
**Pull request by smithaharisankar**
This is the beginning of walnutcreek file.

Please provide feedback.

### CODE REVIEW

File: smitha-Walnutcreek.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Walnut Creek 
+
+### Restaurants
+
+### Local Attractions
+
+sanfrancisco city
```
Comment (Sreeparvathy) [inline smitha-Walnutcreek.md]: is it in Walnut Creek? 👊

Comment (aharden) [inline smitha-Walnutcreek.md]: How about Mt. Diablo?

Comment (smithaharisankar) [inline smitha-Walnutcreek.md]: Added that

**Comment by aharden** [comment]
🚢

**Comment by Sreeparvathy** [comment]
> Added that
>  Add a line note
> View changes

Ready to 🚢

**Pull request merged**